### PR TITLE
Multipath - GET always return map with multiple path args

### PR DESCRIPTION
**Pull request by oshadmi**

### CODE REVIEW

File: src/commands.rs [hunk 1]
```diff
@@ -231,10 +236,10 @@ impl<'a, V: SelectValue> KeyValue<'a, V> {
             return Err("Soon to come...".into());
         }
         let is_legacy = !paths.iter().any(|p| !p.is_legacy());
-        if is_legacy {
-            self.to_json_legacy(paths, indent, newline, space)
+        if paths.len() > 1 {
+            self.to_json_multi(paths, indent, newline, space, is_legacy)
         } else {
-            self.to_json_multi(paths, indent, newline, space)
+            self.to_json_single(paths, indent, newline, space, is_legacy)
```
Comment (MeirShpilraien) [inline src/commands.rs:242]: Why not just passing the single path?

File: tests/pytest/test_multi.py [hunk 2]
```diff
@@ -616,6 +623,14 @@ def testObjKeysCommand(env):
...
     r.assertEqual(res, ["nested1", "a", "nested2"])
 
+    r.assertOk(r.execute_command('JSON.SET', 'doc2', '$', '{"a":{}, "nested":{"a":{"buz":"zer"}}}'))
+    res = r.execute_command('JSON.OBJKEYS', 'doc2', '$..a')
+    r.assertEqual(res, [[], ["buz"]])
+
+    r.assertOk(r.execute_command('JSON.SET', 'doc2', '$', '{}'))
+    res = r.execute_command('JSON.OBJKEYS', 'doc2', '$')
+    r.assertEqual(res, [[]])
+
```
Comment (MeirShpilraien) [inline tests/pytest/test_multi.py:633]: How does those tests related to this PR?

**Review by MeirShpilraien — COMMENTED** [review]
Small comments

File: tests/pytest/test_multi.py (hunk 2 above)
Comment (oshadmi) [reply tests/pytest/test_multi.py]: Right, unrelated.

**Review by MeirShpilraien — APPROVED** [review]

**Pull request merged**